src/hooks/mongoose.js

```diff
@@ -0,0 +1,152 @@
+import { safeRequire } from '../utils/requireUtils';
+import * as logger from '../logger';
+import { hook, hookPromise } from '../extender';
+import { getRandomId, isPromise } from '../utils';
+import { SpansContainer, TracerGlobals } from '../globals';
+import { createMongoDbSpan, extendMongoDbSpan } from '../spans/mongoDbSpan';
+import { getCurrentTransactionId } from '../spans/awsSpan';
+
+const mongooseQueries = [
```

Comment:
BTW, Why our MongoDB instrumentation doesn't work on mongooses?

Reply:
not sure why but it doesnt